### PR TITLE
Package orun.0.1.2

### PR DESCRIPTION
**Pull request by gs0510**
### `orun.0.1.2`
Run benchmarks and measure performance

---
* Source repo: git+https://github.com/ocaml-bench/orun.git

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/orun/orun.0.1.2/opam [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+authors: "Stephen Dolan <stephen.dolan@cl.cam.ac.uk>"
+license: "MIT"
+dev-repo: "git+https://github.com/ocaml-bench/orun.git"
+depends: [ "ocaml" "cmdliner" "yojson" ]
+build: [
+  ["ocaml-update-c" "wait4.c"] {ocaml:update-c}
+  ["dune" "build" "-p" name]
+]
+url {
+  src: "https://github.com/ocaml-bench/orun/archive/0.1.0.tar.gz"
```
Comment (kit-ty-kate) [inline packages/orun/orun.0.1.2/opam]: This does not seem to be `orun.0.1.2`

File: packages/orun/orun.0.1.2/opam [hunk 2]
```diff
@@ -0,0 +1,18 @@
+opam-version: "2.0"
+synopsis: "Run benchmarks and measure performance"
+maintainer: "Stephen Dolan <stephen.dolan@cl.cam.ac.uk>"
+authors: "Stephen Dolan <stephen.dolan@cl.cam.ac.uk>"
+license: "MIT"
+dev-repo: "git+https://github.com/ocaml-bench/orun.git"
+depends: [ "ocaml" "cmdliner" "yojson" ]
+build: [
+  ["ocaml-update-c" "wait4.c"] {ocaml:update-c}
```
Comment (kit-ty-kate) [inline packages/orun/orun.0.1.2/opam]: I'm not sure of what `{<package>:update-c}` means. I don't see this anywhere in the documentation.

File: packages/orun/orun.0.1.2/opam [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  ["dune" "build" "-p" name]
```
Comment (kit-ty-kate) [inline packages/orun/orun.0.1.2/opam]:
```suggestion
  ["dune" "build" "-p" name "-j" jobs]
```
This allows users to select how many processes they want to use to build this package.

**Comment by kit-ty-kate** [comment]
Build fail with:
```
#=== ERROR while compiling orun.0.1.2 =========================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.09.0 | pinned(https://github.com/ocaml-bench/orun/archive/0.1.0.tar.gz)
# path                 ~/.opam/4.09/.opam-switch/build/orun.0.1.2
# command              ~/.opam/4.09/bin/dune build -p orun
# exit-code            1
# env-file             ~/.opam/log/orun-23-322141.env
# output-file          ~/.opam/log/orun-23-322141.out
### output ###
#       ocamlc profiler.o (exit 2)
# (cd _build/default && /home/opam/.opam/4.09/bin/ocamlc.opt -g -ccopt -O2 -ccopt -fno-strict-aliasing -ccopt -fwrapv -ccopt -fPIC -ccopt -g -o profiler.o profiler.c)
# profiler.c:24:10: fatal error: elfutils/libdwfl.h: No such file or directory
#  #include <elfutils/libdwfl.h>
#           ^~~~~~~~~~~~~~~~~~~~
# compilation terminated.
#     ocamldep .orun.eobjs/profiler.ml.d
# File "profiler.ml", line 106, characters 32-34:
# 106 | let slash_regex = Str.regexp "[/\.]"
#                                       ^^
# Warning 14: illegal backslash escape in string.
```

**Comment by gs0510** [comment]
Closing this for now, the package requires some system dependencies. Thanks for the review @kit-ty-kate!

**Pull request closed without merging**